### PR TITLE
Create python base images

### PR DESCRIPTION
**Pull request by dbason**

### CODE REVIEW

File: images/python/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+numpy==1.20.3
+opni-nats==0.0.0.2
+pandas==1.2.3
+aiohttp==3.7.4.post0
+boto3==1.17.45
+botocore==1.20.45
+scikit_learn==0.24.1
+aiohttp==3.7.4.post0
+kubernetes==12.0.1
+schedule
```
Comment (sanjay920) [inline images/python/requirements.txt:11]: can you set this version to `1.1.0`

File: images/python/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+cachetools==4.2.1
+elasticsearch[async]==7.12.0
+jsonpickle==1.5.1
+uvicorn==0.13.4
```
Comment (sanjay920) [inline images/python/requirements.txt:15]: pls sort this file

**Review by sanjay920 — CHANGES_REQUESTED** [review]
minor changes requested

File: images/python/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+FROM builder1 as builder2
+
+COPY ./requirements-torch.txt /requirements-torch.txt
+
+# Work around for problem with docker copy - https://github.com/moby/moby/issues/21950
+RUN find /opt/venv/ -type f > /files-to-delete.txt
+
+RUN pip install -r /requirements-torch.txt
+
+RUN cat /files-to-delete.txt | xargs rm -f
```
Comment (kralicky) [inline images/python/Dockerfile:27]: This might break on files containing spaces, suggest changing to 
`cat /files-to-delete.txt | xargs -d'\n' -- rm -f`

**Review by kralicky — CHANGES_REQUESTED** [review]

**Review by kralicky — APPROVED** [review]

**Review by sanjay920 — APPROVED** [review]

**Pull request merged**